src/components/ConfirmTermination/ConfirmTermination.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { confirm } from '@amsterdam/bmi-component-library';
+import ConfirmTermination from './ConfirmTermination';
+
+const emptyButtons = {
```

Comment:
I'm not sure why you keep referring to callbacks as buttons :thinking: To me this seems like a badly chosen variable name

Reply:
Changed to `callbackMocks`